src/legacy/ui/public/courier/utils/courier_inspector_utils.ts

```diff
@@ -28,7 +28,7 @@ import { i18n } from '@kbn/i18n';
 import { SearchResponse } from 'elasticsearch';
```

Comment:
Seems like changes in this file don't do anything.

Reply:
That's correct, they don't make any functional changes; this was simply to make it faster/easier for me to parse which items are exported and which aren't as I'm making the public NP contract. (The rest of the module uses inline exports like this)